### PR TITLE
Sys tray submenus

### PR DESCRIPTION
**Pull request by roxanneskelly**
Create submenus for People and Goto systray menus.  The Peoplesubmenu includes a list of connections who are online.  The Goto submenu contains a list of featured stories.

https://highfidelity.manuscript.com/f/cases/19205/Tray-Notifier-Stretch-Goals-Submenus

### CODE REVIEW

**Comment by roxanneskelly** [comment]
Test Plan
1) People Submenus
* Launch Console/Sandbox
* Make sure interface is not running
* Log in connection on second machine
* Wait up to 2 minutes
* Notification should pop up (don't click it)
* Validate red light appears next to people
* Validate submenu contains connection
* Click on submenu
* Validate interface starts, and you're teleported to your connection

2) Goto Submenu
* Start console/sandbox
* Stop interface
* Create featured story
* Validate notification comes up (don't click it)
* Validate red light appears next to Goto menu item
* Validate featured story location shows up in Goto submenu

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13782/HighFidelity-Beta-PR14179-0302fee.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13782/HighFidelity-Beta-PR14179-0302fee.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13782/HighFidelity-Beta-Interface-PR14179-0302fee.exe

**Comment by roxanneskelly** [comment]
https://highfidelity.manuscript.com/f/cases/19205/Tray-Notifier-Stretch-Goals-Submenus

**Comment by roxanneskelly** [comment]
[sys tray: submenus](https://trello.com/c/03WnwuMS/689-sys-tray-submenus)

File: server-console/src/modules/hf-notifications.js [hunk 1]
```diff
@@ -357,7 +370,33 @@ HifiNotifications.prototype = {
...
+                                finished(false);
+                                return;
+                            }
+                            var content = JSON.parse(data.body);
+                            if (!content || content.status != 'success') {
+                                console.log("Error: unable to get " + url);
+                                finished(false);
+                                return;
+                            }
+                            console.log(content);
```
Comment (howard-stearns) [inline server-console/src/modules/hf-notifications.js]: do we need this?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13795/HighFidelity-Beta-PR14179-f6a8383.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13795/HighFidelity-Beta-PR14179-f6a8383.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13795/HighFidelity-Beta-Interface-PR14179-f6a8383.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13799/HighFidelity-Beta-PR14179-af055bb.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13802/HighFidelity-Beta-PR14179-6b05f7b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13802/HighFidelity-Beta-PR14179-6b05f7b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13802/HighFidelity-Beta-Interface-PR14179-6b05f7b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13802/HighFidelity-Beta-PR14179-6b05f7b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13802/HighFidelity-Beta-Interface-PR14179-6b05f7b.exe

**Comment by qligmaloney** [comment]
Build this please

**Comment by rickcarr044** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13976/HighFidelity-Beta-PR14179-6b05f7b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13976/HighFidelity-Beta-PR14179-6b05f7b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13976/HighFidelity-Beta-Interface-PR14179-6b05f7b.dmg

**Comment by rickcarr044** [comment]
Currently testing PR14179

**Pull request merged**